build/nv.d3.js

```diff
@@ -12324,7 +12324,10 @@ nv.models.stackedArea = function() {
                 .width(availableWidth)
                 .height(availableHeight)
                 .x(getX)
-                .y(function(d) { return d.display.y + d.display.y0 })
+                .y(function(d) {
+                  if( typeof d.display === 'undefined' ) {}
+                  else { return d.display.y + d.display.y0 }
```

Comment:
> you might need the "typeof" in front of d.display but I'm not sure. 

na, you can just compare the the literal undefined, instead of checking if typeof returns a string of 'undefined'...   a bit cleaner that way imho :)